notes/style-guide.md

```diff
@@ -163,29 +175,27 @@ This is very much a work-in-progress and is not exhaustive.
 
 #### Implicit and explicit arguments
 
-* Functions arguments should be implicit if they can "almost always"
+* Function's arguments should be implicit if they can "almost always"
```

Comment:
Thanks for spotting the typo, I think this should just read for "Function arguments..."